### DISCUSSION TITLE
Add a way to vote for a package?

### DISCUSSION THREAD

**Discussion by Hubro** [Morganamilo/paru#915] [Ideas]
One pain point I have with paru is that I can't vote on packages. I usually have to Google the package name plus "aur" then vote on it in the browser. It would be nice if there was a command in paru for voting on AUR packages.

**Comment by Morganamilo** [comment]
Just use `ssh aur.archlinux.org vote pgkname`

**Reply by Hubro** [reply]
:exploding_head:

**Reply by Hubro** [reply]
Hmm, I couldn't make this work :thinking: Which username do I use? I tried a couple of variants and I suspect I've now been IP blocked, I now get an immediate "Connection refused".

EDIT: Never mind, seems the entire AUR is down. I hope I didn't do that.

EDIT: No wait, what the fuck!? I'm actually IP banned from the AUR, I'm getting connection refused on HTTP, HTTPS and SSH, but I can connect from my remote server :rofl: I hope this is temporary, and I wasn't just permanently banned from using the AUR because of 2-3 failed login attempts.

**Comment by ghost** [comment]
did you figure this out? i do `ssh treeshateorcs@aur.archlinux.org vote cemu` (treeshateorcs is my aur username) and i get Permission denied (publickey). (i did add the public key to the aur settings)

**Comment by ghost** [comment]
ohhh i found it in the wiki, the actual command is `ssh aur@aur.archlinux.org vote pkgname`
